mdoc/Mono.Documentation/Updater/Formatters/CSharpFullMemberFormatter.cs

```diff
@@ -238,7 +238,19 @@ static string GetTypeKind (TypeDefinition t)
             if (t.IsEnum)
                 return "enum";
             if (t.IsValueType)
-                return "struct";
+            {
+                StringBuilder buf = new StringBuilder();
+                if (DocUtils.HasReadOnlyAttribute(t))
+                {
+                    buf.Append("readonly ");
+                }
+                if (t.CustomAttributes.Any(attr => attr.AttributeType.FullName == "System.Runtime.CompilerServices.IsByRefLikeAttribute"))
```

Comment:
Shall we use a method the same as HasReadOnlyAttribute?

Reply:
@yishengjin1413  We can reuse the ReadOnlyAttribute so I extract it to a method. But for this attribute, it only stands for `ref struct`. We are not able to reuse it elsewhere. So I think there's no need to extract it to a method.

Reply:
@yishengjin1413  updated